File: problem_solving_for_beginners/array_problems/c-programming/sum_of_2darray.c
#include<stdio.h>
/**
 * main - Sum of elements of 2D array
 *
 * Return: 0 Always (success)
 */

int main(void)
{
    int n, i, j;
    printf("Enter the size of array: ");
    scanf("%d", &n);

    int arr[n][n];

    for(i = 0; i < n; i++)
    {
        for(j = 0; j < n; j++)
        {
            printf("Enter the element of array: ");
            scanf("%d", &arr[i][j]);
        }
    }
    
    int sum = 0;
    for (i = 0; i < n; i++)
    {
        for (j = 0; j < n; j++)
            sum += arr[i][j];
    }
    
    printf("%d\n", sum);
    return 0;
}

File: problem_solving_for_beginners/array_problems/python/sum_of_2Darray.py
#!/usr/bin/python3
"""Sum of elements of 2D array"""
arr = []
n = int(input("Enter a size of array: "))

for i in range(n):
    col = []
    for j in range(n):
        j = int(input("Enter element of array: "))
        col.append(j)
    arr.append(col)

#write your code here
sum = 0
for i in range(n):
    for j in range(n):
        sum += arr[i][j]

print(sum)